specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -1299,7 +1304,8 @@
           "x-ms-enum": {
             "name": "CreateMode",
             "modelAsString": false
-          }
+          },
+          "x-ms-secret": true
```

Comment:
`createMode` doesn't seem to be a secret?

Reply:
It is for fixing this swagger error: https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=6412D966-7158-4823-A942-17403BA9635C. Based on the document [here](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md#ROUNDTRIP_MISSING_PROPERTY), write-only or secret-carrier can marked with "x-ms-secret"

Reply:
write-only and secret is different things, please use "x-ms-mutability": ["create", "update"] for write-only annotation.

Reply:
@jiacheng-L , can you make this change?